models/recipe-model.js

```diff
@@ -0,0 +1,34 @@
+const mongoose = require("mongoose");
+
+const Schema = mongoose.Schema;
+const recipeSchema = new Schema({
+  title: {type: String, required: true, unique: true},
+  level: {type: String, enum: ["Easy Peasy", "Amateur Chef", "UltraPro Chef"]},
+  ingredient: {type: []},
+  cuisine: {type: String, required: true},
+  dishType: {
+    type: String,
+    enum: ["Breakfast", "Dish", "Snack", "Drink", "Dessert", "Other"]
+  },
+  image: {
+    type: String,
+    default: "https://images.media-allrecipes.com/images/75131.jpg"
+  },
+  duration: {type: String, min: 0},
+  creator: {type: String},
+  created: {type: [Date]}
```

Comment:
You can set a default created time by adding `default : Date.now`

Reply:
I heard nizar saying that it should be made in the code when inserting the value, but it's great news actually ! can it be done the same way with timestamp ?